### DISCUSSION TITLE
Sponsor Request

### DISCUSSION THREAD

**Discussion by Doctor-wu** [community/community#114458] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I need to apply for GitHub Sponsor, which has been under review for more than several months, please help me 🙏.

My Location: HongKong.

GitHub Username: Doctor-wu

**Comment by AidenJamesYt** [comment, marked as answer]
i would open a [Support Ticket](https://support.github.com/) for that.
